user_docs/en/changes.md

```diff
@@ -8,7 +8,7 @@ There are many improvements to speech, particularly responsiveness with SAPI 4,
 Rate boost and automatic language switching is now supported in SAPI 5.
 SAPI 4 voices now support audio ducking, leading silence trimming, and keeping the audio device awake.
 
-The Add-on Store's automatic update system has been improved, allowing you to select channels for automatic updates, and run automatic updates in the background.
+The Add-on Store's update system has been improved, allowing you to select channels for update checks, and run automatic updates in the background.
```

Comment:
Again, I find this change misleading as it doesn't affect the updates tab in the add-on store